mParticle-Kochava/MPKitKochava.m

```diff
@@ -255,10 +259,41 @@ - (MPKitExecStatus *)didFinishLaunchingWithConfiguration:(NSDictionary *)configu
         KVALog.shared.level = [self.configuration[kvEnableLogging] boolValue] ? KVALogLevel.debug : KVALogLevel.never;
     }
     
-    if ([configuration[kvUseCustomerId] boolValue] || [configuration[kvIncludeOtherUserIds] boolValue]) {
+    if ([self.configuration[kvUseCustomerId] boolValue] || [self.configuration[kvIncludeOtherUserIds] boolValue]) {
         [self synchronize];
     }
     
+    NSDictionary *userActivityDictionary = self.launchOptions[UIApplicationLaunchOptionsUserActivityDictionaryKey];
+    if (userActivityDictionary == nil)
+    {
+        [KVADeeplink processWithURL:nil completionHandler:^(KVADeeplink * _Nonnull deeplink)
+         {
+            NSString *destinationString = deeplink.destinationString;
+            if (destinationString.length == 0) {
+                [self->_kitApi onAttributionCompleteWithResult:nil error:[self errorWithMessage:@"Received nil deeplink from Kochava"]];
+                return;
+            }
+            
+            NSMutableDictionary *innerDictionary = [NSMutableDictionary dictionary];
+            if (deeplink.destinationString) {
```

Comment:
Minor nitpick, but now that you have a local variable for `destinationString`, we should probably use it on this line and the line below in place of `deeplink.destinationString`. Or alternatively, just use `deeplink.destinationString.length` directly in the if statement and skip creating the local variable altogether.

Reply:
I had already been debating on removing the `if (deeplink.destinationString) {` since checking the length already covered that. So thanks for tipping the scales.